sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/schema.yaml

```diff
@@ -101,3 +101,54 @@ fields:
 - mode: NULLABLE
   name: tagged_follow_on_search_ad_impressions
   type: INTEGER
+- mode: NULLABLE
```

Comment:
Could we add field descriptions where appropariate?

Reply:
Yeah, this is on my list of things to do 😅 I'll have to revisit this table in a while more holistically anyway